src/components/BubbleLisItem/BubbleListItem.tsx

```diff
@@ -1,33 +1,41 @@
 import React from "react";
-import Image from "next/image"
 import { Bubble } from "@prisma/client";
 import styles from './_bubbleListItem.module.css';
 
-import NoImage from '../../assets/images/no-image.jpg';
-
 type BubbleProps = Bubble & {
   author: {
       avatarUrl: string;
   };
-}
+};
 
 type Props = {
   bubble: BubbleProps,
   onClick: (event: React.MouseEvent<HTMLDivElement, MouseEvent>) => void,
   props?: any
-}
-
-// {authorAvatar && <Image height={'20px'} width={'20px'} src={authorAvatar}/>}
+};
 
 const BubbleListItem: React.FC<Props> = ({ bubble, onClick, ...props }: Props) => {
-  const authorAvatar = bubble ? bubble.author?.avatarUrl : 'https://winaero.com/blog/wp-content/uploads/2019/09/Chrome-Incognito-Mode-Icon-256.png';
+  function Avatar() {
+    const avatar = bubble.author?.avatarUrl;
+  
+    if(avatar == null) {
+      return 'https://winaero.com/blog/wp-content/uploads/2019/09/Chrome-Incognito-Mode-Icon-256.png'
+    };
+  };
+
+  const authorAvatar = Avatar();
+
```

Comment:
daria pra trocar essas 10 linhas por isso:
```suggestion
  const authorAvatar = bubble?.author?.avatarUrl ? bubble.author.avatarUrl : 'https://winaero.com/blog/wp-content/uploads/2019/09/Chrome-Incognito-Mode-Icon-256.png';
```

Reply:
o que acha?
Um jeito melhor ainda seria colocar essa imagem no nosso public. Daí ficaria
```suggestion
  const authorAvatar = bubble?.author?.avatarUrl 
      ? bubble.author.avatarUrl
      : '/anonimous-avatar';
```

Reply:
Certo! Vou testar e se funcionar ja mando um commit